storage/elasticsearch/client.go

```diff
@@ -83,34 +83,43 @@ type ElasticSearchClientInterface interface {
 	Delete(obj string, id string) (elastigo.BaseResponse, error)
 	BulkDelete(obj string, id string)
 	Search(obj string, query string, index string) (elastigo.SearchResult, error)
-	Start(name string, mappings []map[string][]byte, limits ElasticLimits)
+	Start()
 	GetIndexAlias() string
 	GetIndexAllAlias() string
 }
 
+// Mappings describes the mappings of the clinet connection
+type Mappings []map[string][]byte
+
 // ElasticIndex describes an ElasticSearch index and its current status
 type ElasticIndex struct {
 	sync.Mutex
 	entriesCounter int
-	mappings       []map[string][]byte
-	name           string
 	path           string
 	timeCreated    time.Time
-	limits         ElasticLimits
+}
+
+// ConnConfig describes the ElasticSearch configuration section
+type ConnConfig struct {
```

Comment:
Change to ConnectionConfig

Reply:
Using shot names is widly accepted in golang (as opposed for example to Java) srarting from reserved words eg func, chan, int; on to module names eg fmt; and variable names. 

Specifically conn as used widely in idiomatic golang code

Reply:
Yes, I see that in the code in many places (including the lib); so I'm ok with that change ... although very confusing to see conn and think what it stands for, connection is much more understandable.